source/module_io/read_cube.cpp

```diff
@@ -147,43 +140,51 @@ void ModuleIO::trilinear_interpolate(
     delete[] read_rho;
 }
 
-ModuleIO::CubeInfo ModuleIO::read_cube(const std::string& file)
+bool ModuleIO::read_cube(const std::string& file,
+    std::vector<std::string>& comment,
```

Comment:
you do not need to use std::vector to store two lines, one line with '\n' is better.

Reply:
I've changed following your suggestions in the latest comment, except for this. 
If I do so,  it means the caller should ensure there's one and only "\n" in the string passed in, or the number of lines cannot be fixed to 2.
Instead, I add a check that there's no "\n" the first 2 strings.